src/services/AlertService.ts

```diff
@@ -49,7 +49,7 @@ export class AlertService {
   }
 
   public showAlert(config: IAlertModel): Promise<DialogCloseResult> {
-    return this.dialogService.open(Alert, config, { keyboard: true });
+    return this.dialogService.open(Alert, config, { keyboard: true }, config.className);
```

Comment:
_(General comment about dialog behavior)_

@dkent600

There is this specification in the SOT [1]:
> Closing this pop up will navigate to the home page

I'll need to add an "x" to close the modal.
Do this in this PR or follow up okay?
![grafik](https://user-images.githubusercontent.com/30693990/155839162-5b628a7f-caa4-4e91-9bd8-16dc64af6506.png)


[1]: https://www.notion.so/primedao/Registration-Wizard-74901348d4a443a6891ad16fa1c0b42d#eb8411cc81eb4e66b8b86297f375b89a

Reply:
Do not add the X.  It was agreed we will not have that in any modal popups.